AaSystem/EventQueue/CommandQueue.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+import queue
+
+
+commandQueue = "commandQueue"
+tempQueue = "tempQueue"
+
+
+CommandQueues = {
+    "main": {
+        commandQueue: queue.Queue(),
+        tempQueue: queue.Queue()
+    }
+}
+
+
+def CreateNewCommandQueueIfNonExist(queueName):
```

Comment:
change to classes and use constructor

Reply:
Same as previous answer - moving back to single queue